scrapy/pipelines/media.py

```diff
@@ -16,6 +19,7 @@
 class MediaPipeline(object):
 
     LOG_FAILED_RESULTS = True
+    ALLOW_REDIRECTS = False
```

Comment:
Looks like answer is No...http://stackoverflow.com/questions/13592653/multi-user-github-pull-requests

@Granitosaurus would have to accept a merge from me then it would chain through his PR